desktop-widgets/mapwidget.cpp

```diff
@@ -22,19 +28,23 @@ MapWidget::MapWidget(QWidget *parent) : QQuickWidget(parent)
 	qmlRegisterType<MapLocationModel>("org.subsurfacedivelog.mobile", 1, 0, "MapLocationModel");
 	qmlRegisterType<MapLocation>("org.subsurfacedivelog.mobile", 1, 0, "MapLocation");
 
+	setResizeMode(QQuickWidget::SizeRootObjectToView);
 	connect(this, &QQuickWidget::statusChanged, this, &MapWidget::doneLoading);
-	setSource(QUrl(QStringLiteral("qrc:/MapWidget.qml")));
+	setSource(QUrl(urlMapWidget));
```

Comment:
urlMapWidget is a QUrl - so why are you creating a new QUrl from it?

Reply:
i will fix this.